backup-import/src/main/java/com/opsgenie/tools/backup/importers/PolicyImporter.java

```diff
@@ -148,8 +149,15 @@ protected void updateTeamIds(PolicyWithTeamInfo entity) throws Exception {
             if(entity.getPolicy().getClass() == AlertPolicy.class) {
                 AlertPolicy alertPolicy = (AlertPolicy) entity.getPolicy();
                 for(Responder responder : alertPolicy.getResponders()) {
-                    if(responder.getId().equals(entity.getTeamId())) {
-                        responder.setId(teamIdMap.get(teamName));
+                    if(responder.getType() == Responder.TypeEnum.TEAM) {
+                        File teamsDirectory = new File(backupRootDirectory + "/teams/");
+                        String responderTeamName = BackupUtils.getTeamNameFromId(teamsDirectory, responder.getId());
+                        if (responderTeamName != null){
```

Comment:
i have a clarifying question is responderTeamName is `string`? If yes, can we use StringUtils.empty(responderTeamName) otherwise we are good.